rust-toolchain.toml

```
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly"
+components = ["rustfmt", "clippy"]
```

I think this should be sufficient:

```suggestion
[toolchain]
channel = "1.88.0"
components = ["rustfmt", "clippy"]
```